src/app/dynamic_server/AccessControl.h

```diff
@@ -19,6 +19,7 @@
 
 constexpr chip::EndpointId kOtaProviderDynamicEndpointId     = 0;
 constexpr chip::EndpointId kWebRTCRequesterDynamicEndpointId = 1;
+constexpr chip::EndpointId kWebRTCProviderDynamicEndpointId  = 1;
```

Comment:
Is it ok to put the requestor and provider on the same endpoint?

Reply:
Thank you for your review. Yes, it should not be a problem.